releases/v22.1.md

```diff
@@ -27,3 +28,7 @@ To upgrade to {{ page.major_version }}, see [Upgrade to CockroachDB {{ page.majo
 {% for r in rel %}
 {% include releases/{{ page.major_version }}/{{ r.version }}.md release=r.version %}
 {% endfor %}
+
+{% else %}
+Currently, no releases are available for this version.
```

Comment:
```suggestion
No releases are available for this version. See the [Releases](https://www.cockroachlabs.com/docs/releases/index.html) page for all available releases.
```

May as well link out, I suppose, though this should be sufficient either way.

Reply:
Agreed. Done.